Jenkinsfile.deploy

```diff
@@ -122,30 +141,20 @@ pipeline {
             }
             steps {
                 script {
-                    mavenCleanInstallOptaPlannerParents('optaplanner')
-                    dir('optaplanner') {
-                        maven.mvnVersionsSet(getProjectVersion())
-                        updateKogitoVersion(getKogitoVersion())
-                    }
-                    mavenCleanInstallOptaPlannerParents('optaplanner')
-                    dir('optaweb-vehicle-routing') {
-                        maven.mvnVersionsUpdateParentAndChildModules(getProjectVersion())
-                    }
-                    dir('optaweb-employee-rostering') {
-                        maven.mvnVersionsUpdateParentAndChildModules(getProjectVersion())
-                    }
+                    maven.mvnVersionsSet(defaultOptaplannerMvnCmd, getProjectVersion())
+                    maven.mvnSetVersionProperty(defaultOptaplannerMvnCmd, 'version.org.kie.kogito', getKogitoVersion())
+                    defaultOptaplannerMvnCmd.clone().skipTests(true).withOptions(['-U']).run('clean install')
```

Comment:
The `mavenCleanInstallOptaPlannerParents` ran only specific modules (a parent pom.xml and BOM) the other projects depend upon to save time (a couple of seconds vs. ~5 minutes).
I don't expect these two modules to be renamed; especially the BOM is a part of our API.

Reply:
woops. gonna correct that

Reply:
updated